### PR TITLE
Interfaces

### PR DESCRIPTION
**Pull request by CDuPlooy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,10 +25,10 @@
     "@types/chartjs": "0.0.31",
     "angular-font-awesome": "^3.1.2",
     "angular2-jwt": "^0.2.3",
-    "cerialize": "^0.1.18",
     "chart.js": "^2.7.2",
     "core-js": "^2.4.1",
     "font-awesome": "^4.7.0",
+    "jsdoc": "^3.5.5",
```
Comment (egeldenhuys) [inline package.json]: Can we make jsdoc a dev dependency?

File: src/app/_models/container/mount/mount.model.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/**
+ * Interface for Mount belonging to Container
+ *
+ * @interface
+ */
+export interface Mount {
+    Name: string;
```
Comment (egeldenhuys) [inline src/app/_models/container/mount/mount.model.ts:7]: Unexpected upper case property names. Lower case is preferred.
See [style-03-03](https://angular.io/guide/styleguide#style-03-03) and [style-03-04](https://angular.io/guide/styleguide#style-03-04)

File: docks-ui.iml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
```
Comment (egeldenhuys) [inline docks-ui.iml]: What is this file for?

**Review by egeldenhuys — CHANGES_REQUESTED** [review]
:+1: The interfaces read much better than the classes using cerialize.

:+1: The models appear to be complete, although I have not worked with them yet

:+1: Absolute paths for importing models

**Review by annamarieHelberg — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (CDuPlooy) [reply package.json]: whoops, only saw this now. sure

File: src/app/_models/container/mount/mount.model.ts (hunk 2 above)
Comment (CDuPlooy) [reply src/app/_models/container/mount/mount.model.ts:7]: As far as I know the property names are required to be the same as the json that its parsing. So I used uppercase due to the api returning uppercase property names.

File: docks-ui.iml (hunk 3 above)
Comment (CDuPlooy) [reply docks-ui.iml]: I have no idea, I thought you added this xDD I'll remove it.

File: src/app/_models/container/mount/mount.model.ts (hunk 2 above)
Comment (egeldenhuys) [reply src/app/_models/container/mount/mount.model.ts:7]: I did not know that, then it makes sense. Thank you for clarifying that.

**Review by egeldenhuys — APPROVED** [review]

**Pull request merged**